Patches/Moyo light in the abyss/ThingDefs_Misc/Moyo_Apparel.xml

```diff
@@ -0,0 +1,580 @@
+					<Bulk>3</Bulk>
+					<WornBulk>2</WornBulk>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="RedMoyo_WorkSuit"]/equippedStatOffsets</xpath>
+				<value>
+					<CarryBulk>5</CarryBulk>
+				</value>
+			</li>
+			
+			
+			<!--========= Fixed Armor Value =========-->
+			
+			<!--Bodysuit-->
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="RedMoyo_BobySuit"]/statBases</xpath>
+				<value>
+					<Bulk>5</Bulk>
+					<WornBulk>2</WornBulk>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RedMoyo_BobySuit"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>1.5</ArmorRating_Sharp>
+					<ArmorRating_Electric>0.15</ArmorRating_Electric>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RedMoyo_BobySuit"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>0.3</ArmorRating_Blunt>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="RedMoyo_BobySuit"]/apparel/bodyPartGroups</xpath>
+				<value>
+					<li>Hands</li>
+					<li>Feet</li>
+				</value>
+			</li>
+			
+			<!--Tactical Strap-->
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="RedMoyo_TacticalStrap"]/statBases</xpath>
+				<value>
+					<Bulk>5</Bulk>
+					<WornBulk>2</WornBulk>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RedMoyo_TacticalStrap"]/statBases/MaxHitPoints</xpath>
+				<value>
+					<MaxHitPoints>175</MaxHitPoints>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RedMoyo_TacticalStrap"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>2.5</ArmorRating_Sharp>
+					<ArmorRating_Electric>0.15</ArmorRating_Electric>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="RedMoyo_TacticalStrap"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>3.75</ArmorRating_Blunt>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="RedMoyo_TacticalStrap"]/equippedStatOffsets</xpath>
+				<value>
+					<CarryBulk>20</CarryBulk>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="RedMoyo_TacticalStrap"]/equippedStatOffsets/AimingDelayFactor</xpath>
+				<value>
+					<ReloadSpeed>0.1</ReloadSpeed>
+					<AimingDelayFactor>-0.05</AimingDelayFactor>
+				</value>
+			</li>
+			
+			<!--Police Uniform-->
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="RedMoyo_PoliceUniform"]/statBases</xpath>
+				<value>
+					<Bulk>2</Bulk>
+					<WornBulk>1</WornBulk>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">

```

Comment:
I don't have a good point of reference for how the base mod treats them, but a few items such as the police uniform and cap seem to be more a fabric-like apparel item versus armor. Though, if the base mod gives them an armor value of any significance, 1.5mm is probably fairly reasonable.

Reply:
Police uniform is pretty much equivalent to vanilla flak jacket, and does also need steel. I'll drop the blunt protection a bit though to be closer to flak.